### PR TITLE
Task contacts

### PR DESCRIPTION
**Pull request by Yas-Shen**
I believe I have finished for now. Buttons could do with setting side to side but since they are centred, I've had enough of moving buttons.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -133,19 +135,55 @@ div.project-item {
...
+	justify-content: center;
+	flex-direction: column;
+	background-color: #5911a4;
+	height: 50px;
+	border-radius: 15px;
+	margin-top: 23%;
+}
+
+h4 {
+	display: flex;
```
Comment (nesclark) [inline styles.css:166]: is display: flex needed on h4? as you already have in place for parent container (.contacts)

File: index.html [hunk 2]
```diff
@@ -84,5 +84,14 @@ <h2>Project</h2>
 	</div>
 </div>
 </section>
+
+	<footer>
+		<div class="contacts">
+			<h4>To Contact:-</h4>
```
Comment (nesclark) [inline index.html:90]: this pops out of the contacts bar when resizing browser (not sure why, sorry!)

**Review by nesclark — COMMENTED** [review]
Looks great! love the background and general layout :)

File: index.html (hunk 2 above)
Comment (HadertS) [reply index.html:90]: This seems to be caused by the media queries in styles.css lines 220-236. In my testing removing this fixed the issue, but there may be other knock on effects.

```
@media screen and (max-width: 900px) {
	
	h4 {
		margin: 5% 0 0 0;
	}
	
	.contacts {
		margin: 0 0 4% 0;
	}
}

@media screen and (max-width: 850px) {
	
	footer a {
		margin: 4% 0 0 0;
	}
}

**Pull request merged**